nominees/berkely-air-sums.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "Berkely Air Sums",
```

Comment:
@ijeomaemeruwa kindly correct the spelling for the name field to match the [website](https://berkeleyair.com/). It appears to be `Berkeley` not `Berkely`. Please clarify.